From 7f05a26aee756c39dd37e0b112ab274e6873b906 Mon Sep 17 00:00:00 2001
From: Jan Pazdziora <adelton@fedoraproject.org>
Date: Sat, 10 Aug 2024 23:26:54 +0200
Subject: [PATCH] Addressing widen is not a member of boost::nowide
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 src/slic3r/GUI/UserAccountCommunication.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/slic3r/GUI/UserAccountCommunication.cpp b/src/slic3r/GUI/UserAccountCommunication.cpp
index 405cf55f12b..a32ebfc904d 100644
--- a/src/slic3r/GUI/UserAccountCommunication.cpp
+++ b/src/slic3r/GUI/UserAccountCommunication.cpp
@@ -13,6 +13,7 @@
 #include <boost/filesystem.hpp>
 #include <boost/nowide/cstdio.hpp>
 #include <boost/nowide/fstream.hpp>
+#include <boost/nowide/convert.hpp>
 #include <curl/curl.h>
 #include <string>
 
