diff -up PrusaSlicer-version_2.6.0/src/libslic3r/CMakeLists.txt.no-cereal-lib PrusaSlicer-version_6.4.0/src/libslic3r/CMakeLists.txt
--- PrusaSlicer-version_6.4.0/src/libslic3r/CMakeLists.txt.no-cereal-lib	2022-02-11 20:07:36.399806933 +0000
+++ PrusaSlicer-version_6.4.0/src/libslic3r/CMakeLists.txt	2022-02-11 20:14:53.806895641 +0000
@@ -490,7 +490,6 @@
 target_link_libraries(libslic3r
     libnest2d
     admesh
-    libcereal
     libigl
     miniz
     boost_libs
diff -up PrusaSlicer-version_2.6.0/src/CMakeLists.txt.no-cereal-lib PrusaSlicer-version_2.6.0/src/CMakeLists.txt
--- PrusaSlicer-version_2.6.0/src/CMakeLists.txt.no-cereal-lib	2022-02-11 20:31:04.581810442 +0000
+++ PrusaSlicer-version_2.6.0/src/CMakeLists.txt	2022-02-11 20:31:15.726430666 +0000
@@ -147,7 +147,7 @@
     set_target_properties(PrusaSlicer PROPERTIES OUTPUT_NAME "prusa-slicer")
 endif ()
 
-target_link_libraries(PrusaSlicer libslic3r libcereal)
+target_link_libraries(PrusaSlicer libslic3r)
 
 if (APPLE)
 #    add_compile_options(-stdlib=libc++)
diff -up PrusaSlicer-version_2.6.0/src/slic3r/CMakeLists.txt.no-cereal-lib PrusaSlicer-version_2.6.0/src/slic3r/CMakeLists.txt
--- PrusaSlicer-version_2.6.0/src/slic3r/CMakeLists.txt.no-cereal-lib	2022-02-11 20:30:36.653762157 +0000
+++ PrusaSlicer-version_2.6.0/src/slic3r/CMakeLists.txt	2022-02-11 20:30:47.604388990 +0000
@@ -319,7 +319,7 @@
 
 encoding_check(libslic3r_gui)
 
-target_link_libraries(libslic3r_gui libslic3r avrdude libcereal imgui GLEW::GLEW OpenGL::GL hidapi libcurl ${wxWidgets_LIBRARIES} NanoSVG::nanosvg NanoSVG::nanosvgrast)
+target_link_libraries(libslic3r_gui libslic3r avrdude imgui GLEW::GLEW OpenGL::GL hidapi libcurl ${wxWidgets_LIBRARIES} NanoSVG::nanosvg NanoSVG::nanosvgrast)
 
 if (MSVC)
     target_link_libraries(libslic3r_gui Setupapi.lib)
